### DISCUSSION TITLE
Login?

### DISCUSSION THREAD

**Discussion by TornaxO7** [spikecodes/libreddit#640] [Q&A] [answered]
Hello! I'd like to ask if it's possible to login to reddit through libreddit to see my the subreddits which I joined.

**Comment by spikecodes** [comment, marked as answer]
Hi! Please see [this answer](https://github.com/libreddit/libreddit/wiki/Frequently-Asked-Questions#will-libreddit-support-signing-into-a-reddit-account) in the FAQ Wiki page.

**Reply by TornaxO7** [reply]
Thank you for the link!
